docker/openmrs-build/README.md

```diff
@@ -0,0 +1,50 @@
+# OpenMRS Image Builder
+
+This directory contains a multi-staged `Dockerfile` to build two OpenMRS images:
+
+* An empty OpenMRS image
+* An OpenMRS image with the OCL ZIP file
+
+The empty OpenMRS image is used in the [`openmrs-compose.yaml`](../openmrs-compose.yaml) YAML
+
+The OCL image is used in the [`mysql-init-compose.yaml`](../mysql-build/mysql-init-compose.yaml) to setup the custom SQL image
+
+## PRE-REQ
+
+* MySQL instance running on host machine, listening to port 3306
+* Docker running on host machine
+
+## HOW TO RUN
+
+### Empty OpenMRS image builder
+
+This build a Docker image tagged as:
+`$REPO_NAME/openmrs-openmrs-reference-application-distro:empty`
```

Comment:
Do we actually need this artifact? It seems it is only referenced in `openmrs-compose.yaml` below but it feels we can use the OCL version there too (why not?).